com.reprezen.swagedit.dependencies/pom.xml

```diff
@@ -53,22 +78,71 @@
 						<goals>
 							<goal>copy-dependencies</goal>
 						</goals>
+						<configuration>
+							<outputDirectory>${project.basedir}/lib</outputDirectory>
+							<overWriteReleases>false</overWriteReleases>
+							<overWriteSnapshots>true</overWriteSnapshots>
+							<includeTypes>jar</includeTypes>
+							<prependGroupId>true</prependGroupId>
+							<stripVersion>true</stripVersion>
+							<excludeArtifactIds>com.reprezen.swagedit.dependencies</excludeArtifactIds>
+							<excludeGroupIds>
+								com.google.guava,
+								javax.mail,
+								javax.activation,
+								com.sun.mail
+							</excludeGroupIds>
+							<excludeScope>system</excludeScope>
+							<includeScope>runtime</includeScope>
```

Comment:
I see that this `<configuration>` was pushed down from `<plugin>` level to `<execution>` level, presumably because you added a new execution with its own separate configuration. 

Also, the configuration new has `<excludeScope>` and `<includeScope>` properties. Can you explain these?

Reply:
The `<excludeScope>` and `<includeScope>` are there to prevent eclipse dependencies pulled by tycho to be considered.